### PR TITLE
Add lenovo-thinkpad-p1

### PR DESCRIPTION
**Pull request by alsvartr**
Add lenovo-thinkpad-p1 support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,8 @@ See code for all available configurations.
 | Lenovo ThinkPad L13               | `<nixos-hardware/lenovo/thinkpad/l13>`             |
 | Lenovo ThinkPad L14 (Intel)       | `<nixos-hardware/lenovo/thinkpad/l14/intel>        |
 | Lenovo ThinkPad L14 (AMD)         | `<nixos-hardware/lenovo/thinkpad/l14/amd>          |
+| Lenovo ThinkPad P1								| `<nixos-hardware/lenovo/thinkpad/p1/>`             |
```
Comment (Mic92) [inline README.md:107]: nitpick: This looks like tabs instead of spaces.

File: lenovo/thinkpad/p1/default.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  hardware.nvidia.prime = {
+    # Bus ID of the Intel GPU.
+    intelBusId = lib.mkDefault "PCI:0:2:0";
+
+    # Bus ID of the NVIDIA GPU.
+    nvidiaBusId = lib.mkDefault "PCI:1:0:0";
+  };
+
+	services.fprintd.enable = true;
```
Comment (Mic92) [inline lenovo/thinkpad/p1/default.nix:16]:
```suggestion
  services.fprintd.enable = true;
```

File: README.md (hunk 1 above)
Comment (Mic92) [reply README.md:107]: Please also add the module to `flake.nix

Comment (Mic92) [reply README.md:107]: Did you forgot to push? I don't see the whitespace in the README and `flake.nix` is also unchanged.

Comment (alsvartr) [reply README.md:107]: done

**Pull request merged**